modules/crm_core_match/crm_core_match.test.inc

```diff
@@ -18,148 +18,148 @@ function crm_core_match_testing_page_title($contact) {
  *
  * @return string
  */
-function crm_core_match_testing_page($contact) {
-  $output = '';
-  $engines = crm_core_match_get_engines();
+//function crm_core_match_testing_page($contact) {
```

Comment:
Dropped the **crm_core_match.test.inc** file. I'll paste the code to the new issue.